packages/nc-gui/components/project/Table.vue

```diff
@@ -143,6 +143,7 @@ export default {
             key: this.deleteId,
             value: null
           })
+         this.$toast.info(`Deleted table ${this.nodes.table_name} successfully`).goAway(3000)
```

Comment:
We should use alias/title for consistency

Reply:
Good Point. `table_name` would have the prefix for xcdb projects. Updated now.